openshift/setup-teardown/params/CUSTOM_SETTINGS/PR/environment.config

```diff
@@ -1,4 +1,4 @@
-TOOLS_PROJECT=esm
-TARGET_PROJECT=esm-dev
+TOOLS_PROJECT=md4m71-tools
```

Comment:
Is this correct?

Reply:
good catch, my branches probably need updating